langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -66,6 +101,17 @@ public isolated function abs(decimal x) returns decimal = @java:Method {
 # roundToIntegralTiesToEven will return a value with a positive exponent when the operand has a positive exponent.
 # Note that `<int>x` is the same as `<int>x.round(0)`.
 #
+# ```ballerina
+# decimal d = 3.55;
```

Comment:
Shall we add two cases to show half even mode?
eg: 2.5 to 2 and 3.5 to 4

Reply:
Updated